src/build.sh

```diff
@@ -37,16 +37,12 @@ elif [ $RELEASE_BUILD -eq 0 ]; then
 fi
 pushd tmx
 cmake -Bbuild -DCMAKE_PREFIX_PATH=\"/usr/local/share/tmx\;\/opt/carma/cmake\;\" -DCMAKE_CXX_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_C_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .
-pushd build
-make -j${numCPU}
-make install
-popd
+cmake --build build
```

Comment:
Shall we be consistent with the syntax either all cmake -Bbuild or cmake --build build

Reply:
Good call